core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,39 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
```

Comment:
@seanlip i actually had to remove this comment as i had already moved to exploration player file. If we were to make it globally, then there was unexpected behavior on search bar in library page.